code/modules/mob/living/simple_animal/guardian/guardian.dm

```diff
@@ -40,10 +40,18 @@ var/global/list/parasites = list() //all currently existing/living guardians
 	var/playstyle_string = "<span class='holoparasite'>You are a standard Guardian. You shouldn't exist!</span>"
 	var/magic_fluff_string = "<span class='holoparasite'>You draw the Coder, symbolizing bugs and errors. This shouldn't happen! Submit a bug report!</span>"
 	var/tech_fluff_string = "<span class='holoparasite'>BOOT SEQUENCE COMPLETE. ERROR MODULE LOADED. THIS SHOULDN'T HAPPEN. Submit a bug report!</span>"
+	var/carp_fluff_string = "<span class='holoparasite'>CARP CARP CARP SOME SORT OF HORRIFIC BUG BLAME THE CODERS CARP CARP CARP</span>"
 
 /mob/living/simple_animal/hostile/guardian/New(loc, theme)
 	parasites |= src
 	setthemename(theme)
+
+	//Special case holocarp, because #snowflake code
+	if(theme == "carp")
```

Comment:
Can't be a child because of all the types; it'd be better to add these(icon/emote/desc) to the theme datum and setthemename()
Which would allow for a lot more parasite types, too.